src/PerfView/Authentication.cs

```diff
@@ -183,19 +186,19 @@ private void UpdateState(AuthProviderFlags newState, string propertyName)
     }
 
     /// <summary>
-    /// Extension methods for <see cref="SymbolReaderHttpHandler"/>.
+    /// Extension methods for <see cref="SymbolReaderAuthenticationHandler"/>.
     /// </summary>
     internal static class SymbolReaderHttpHandlerExtensions
```

Comment:
Rename to `SymbolReaderAuthenticationHandlerExtensions`

Reply:
Good catch.